MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/AdsImaSDKListener.kt

```diff
@@ -181,6 +181,17 @@ class AdsImaSDKListener private constructor(
    */
   private fun dispatchAdPlaybackEvent(event: MuxAdEvent, ad: Ad?) {
     setupAdViewData(event, ad)
+
+    if (event.type == AdBreakEndEvent.TYPE) {
```

Comment:
I'm a little confused by this... shouldn't this say `event.type == AdBreakStartEvent.TYPE`?

Reply:
Good spot, you are totally right!